### PR TITLE
Feature: AI chat

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

File: src/test/java/com/ditrit/letomodelizerapi/cucumber/StepDefinitions.java [hunk 1]
```diff
@@ -423,6 +431,11 @@ public void cleanLibrary(String url) throws URISyntaxException, IOException, Int
         this.clean("libraries", String.format("url=%s", url));
     }
 
+    @And("I clean aiConversation {string}")
```
Comment (aboubakar14) [inline src/test/java/com/ditrit/letomodelizerapi/cucumber/StepDefinitions.java:434]: I clean the AI conversation

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -42,6 +42,12 @@ Once database is setup you can run :
 
 > :warning: **You have to generate keystore.jks before!**, see [README](https://github.com/ditrit/leto-modelizer-api/blob/main/README.md#Generate-certificate-for-HTTPS).
 
+If you do not want to have a real ia, you can use a fake ia:
```
Comment (aboubakar14) [inline CONTRIBUTING.md:45]: AI !!!!!! :anger:

Comment (Zorin95670) [reply CONTRIBUTING.md]: done

File: src/test/java/com/ditrit/letomodelizerapi/cucumber/StepDefinitions.java (hunk 1 above)
Comment (Zorin95670) [reply src/test/java/com/ditrit/letomodelizerapi/cucumber/StepDefinitions.java]: done

File: src/main/java/com/ditrit/letomodelizerapi/config/Constants.java [hunk 3]
```diff
@@ -26,6 +26,14 @@ public final class Constants {
      */
     public static final String DEFAULT_USER_PROPERTY = "login";
 
+    /**
+     * Constant defining the default property name used to identify the user within the application.
+     * This property, typically "login", is used as a key in various contexts such as session management,
+     * authentication processes, and user identification within the application's data layer. It represents
+     * the standard field through which users are uniquely identified across the application.
+     */
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply src/main/java/com/ditrit/letomodelizerapi/config/Constants.java]: done

**Review by aboubakar14 — APPROVED** [review]

**Review by aboubakar14 — APPROVED** [review]

**Pull request merged**